src/index.css

```diff
@@ -411,3 +411,8 @@ div.monokleEditorErrorRefGlyphClass.monokleEditorUnsatisfiedRefGlyphClass::after
 .filters-preset-dropdown .ant-select-item-option-content:hover .anticon {
   display: block;
 }
+
+/*temporary*/
+.libes__A6-dev__sc-akj29i-8.gOklpn:last-child {
```

Comment:
Why do we need this? It seems like there are some dynamic classes used here and I'm not sure this will always be working

Reply:
I am hiding the left navigation item the way I don't have to change whole functionality for now. I have to work on this task further with that temporary navigation tab. So decided to temporarily hide it this way. Will remove this later. @topliceanurazvan